doc/source/shark.rst

```diff
@@ -0,0 +1,128 @@
+****************************
+Shark (pcap/tcpdump parsing)
+****************************
+
+Caproto includes a Python function for parsing and analyzing Channel Access
+network traffic in Python. It consumes pcap format, as produced by ``tcpdump``.
+This functionality is also accessible though a CLI, ``caproto-shark``.
+
+.. info::
+
+   This feature of caproto uses an external Python library
+   `dpkt <https://dpkt.readthedocs.io/en/latest/>`_ to parse the pcap format
+   output by ``tcpdump``. A "minimal" installation of caproto---as in
+   ``pip install caproto``---does not include ``dpkt``. Any of the following
+   will install it:
+
+   .. code-block:: bash
+
+      # Any one of these is sufficient:
+      pip install dpkt  # just dpkt
+      pip install caproto[standard]  # all of caproto's network-related extras
+      pip install caproto[complete]  # all of caproto's extras
+
+Capture network traffic to a using ``tcpdump`` like so:
+
+.. code-block:: bash
+
+   sudo tcpdump -w some_network_traffic.pcap port 5064
+
+Extract the information in Python using caproto:
+
+.. code-block:: python
+
+   from caproto.sync.shark import shark
+
+   file = open('some_network_traffic.pcap', 'rb')
```

Comment:
```suggestion
   with open('some_network_traffic.pcap', 'rb') as file:
       parsed = shark(file)
```

Reply:
that is not going to apply well due to not removing the line below...

Reply:
I typed that first, but I think it could lead to someone trying to use parsed after the file had closed.

Reply:
Nevermind, it's easy enough to guide people toward how to do this right. Change pushed.